packages/gateway-client/src/worker/index.ts

```diff
@@ -465,25 +553,7 @@ async function main() {
             },
         },
     });
-    // Listen for new peers
-    //let foundServer = false;
-    node.addEventListener('peer:discovery', evt => {
-        const peer = evt.detail;
-        console.log(`Found peer ${peer.id.toString()}`);
```

Comment:
I'm thinking we should keep this console log.

Reply:
ok, I'll add it back